src/components/Layout.js

```diff
@@ -68,12 +74,18 @@ class Layout extends React.Component {
               <Breadcrumb data={{ title: 'HOME', pathname: MainRoutes.Home }}>
                 <div className='main' data-testid='main'>
                   <Switch>
-                    {/*prettier-ignore*/}
                     <Route component={Home} exact path={MainRoutes.Home} />
-                    <Route component={Apps} path={AppsRoutes.Home} />
+
+                    {user.type === LoggedInUserTypes.MAPI &&
+                    provider !== Providers.KVM ? (
+                      <Route component={AppsMAPI} path={AppsRoutes.Home} />
+                    ) : (
+                      <Route component={Apps} path={AppsRoutes.Home} />
+                    )}
```

Comment:
Created 2 separate components to get ready for https://github.com/giantswarm/giantswarm/issues/17361